tensorboard/writer/event_file_writer_test.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+# if __name__ == '__main__':
+#   return
+
+# """Integration tests for the Writer."""
+
+# from __future__ import absolute_import
+# from __future__ import division
+# from __future__ import print_function
+
+# import collections
+# import os.path
+
+# import six
+# from six.moves import xrange  # pylint: disable=redefined-builtin
+# import tensorflow as tf
+
+# from tensorboard.backend.event_processing import plugin_event_accumulator as event_accumulator  # pylint: disable=line-too-long
+# from tensorboard.backend.event_processing import plugin_event_multiplexer as event_multiplexer  # pylint: disable=line-too-long
+# from tensorboard.plugins import base_plugin
+# from tensorboard.plugins.histogram import histograms_plugin
+# from tensorboard.plugins.histogram import summary
+# from tensorboard.util import test_util
+
+# tf.compat.v1.disable_v2_behavior()
+
+import tensorflow as tf
```

Comment:
If you keep this make sure the test deps in BUILD includes `"//tensorboard:expect_tensorflow_installed",`